source/Camera/Maintenance/CorrectCameraTrueness/CorrectCameraTrueness.cpp

```diff
@@ -0,0 +1,120 @@
+/*
+This example shows how to perform In-field correction. This involves gathering data from
+a compatible Zivid in-field calibration object at several distances, calculating an
```

Comment:
Sorry, I may have suggested to use object instead of board. However, now I see we use board in all our other documentation etc. so please change 'object' to 'board'

Reply:
Will do